### DISCUSSION TITLE
springboot app in docker question

### DISCUSSION THREAD

**Discussion by decadestory** [apache/skywalking#10908] [Q&A]
my springboot app deploy in docker and set docker network=host, skywlking8.13.0 get two ips,172.17.0.1 and 192.168.4.151
skywalking ui instance show like this: 86d26917500546048e385373817aa0ac@172.17.0.1
question:
register instance ok , but there is no data in ........?

can you help me  thank you!

**Comment by decadestory** [comment]
any body?

**Comment by wu-sheng** [comment]
You didn't describe anything. What do you mean `.....`? We can't do anything if you are even not willing to describe the question.
And you ask for `any body`? I can't see the logic.
